src/components/redux/configureStore.js

```diff
@@ -0,0 +1,8 @@
+import { combineReducers, configureStore } from '@reduxjs/toolkit';
+import reducerBooks from './books/books';
+import reducerStatus from './categories/categories';
+
+const rootReducer = combineReducers({ books: reducerBooks, status: reducerStatus });
+const store = configureStore({ reducer: rootReducer });
+
```

Comment:
Hello @HAMMAS-SALEEM,

Thanks for reviewing the PR. It is working with the current change. However, the suggested changes won't work as I am using combineReducer which requires a reducer keyword. To be specific I was getting below error in the console

`Uncaught Error: "reducer" is a required argument, and must be a function or an object of functions that can be passed to combineReducers`

Reply:
Hey @RiyaBulia12 :wave:

I hope you are well. I have checked it and you are right. You can add it as an object as you were doing before. I apologize because i thought it works as same as createStore. You can skip this change dear :blush: